lib/crewai/src/crewai/cli/authentication/providers/base_provider.py

```diff
@@ -28,3 +28,6 @@ def get_client_id(self) -> str: ...
     def get_required_fields(self) -> list[str]:
         """Returns which provider-specific fields inside the "extra" dict will be required"""
         return []
+
+    def get_scope(self) -> str:
+        return "openid profile email "
```

Comment:
Is the extra space at the end intentional?

Reply:
I'm sold! Check out [287e1cd](https://github.com/crewAIInc/crewAI/pull/3943/commits/287e1cd46a49c6d57ec138a9dc64409755fca2a1)